### PR TITLE
UIOR-852 Select Acquisition method from controlled vocabulary list

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

https://issues.folio.org/browse/UIOR-852
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: translations/ui-orders/en.json [hunk 1]
```diff
@@ -6,15 +6,6 @@
   "deletePiece.delete.success": "Selected pieces were deleted",
   "deletePiece.delete.fail": "Selected pieces were not deleted",
   "deletePiece.delete.thereAreRequestsOnItem": "Selected pieces were not deleted. There are requests on item",
-  "acquisition_method.approvalPlan": "Approval plan",
```
Comment (NikitaSedyx) [inline translations/ui-orders/en.json:9]: hm, not sure that default should be always in English, please check how it's done with close reasons

mb point for Dennis

Comment (usavkov-epam) [reply translations/ui-orders/en.json:9]: Yes, the reasons are using translations, I'll make some updates, thanks

Comment (usavkov-epam) [reply translations/ui-orders/en.json]: I updated `getAcqMethodsOptions`, now it returns options with translated default acq methods

File: src/common/POLFields/DetailsFields/FieldAcquisitionMethod.js [hunk 2]
```diff
@@ -3,18 +3,13 @@ import PropTypes from 'prop-types';
...
 
-const ACQUISITION_METHOD_OPTIONS = Object.keys(ACQUISITION_METHOD).map((key) => ({
-  labelId: `ui-orders.acquisition_method.${key}`,
-  value: ACQUISITION_METHOD[key],
-}));
-
-const FieldAcquisitionMethod = ({ disabled, required }) => (
-  <FieldSelectFinal
-    dataOptions={ACQUISITION_METHOD_OPTIONS}
+const FieldAcquisitionMethod = ({ acquisitionMethods, disabled, required }) => (
```
Comment (AndreiShumski2) [inline src/common/POLFields/DetailsFields/FieldAcquisitionMethod.js:9]: seems acquisitionMethods could be fetched inside `FieldAcquisitionMethod` component using react-query and avoid props drilling. what do you think?

Comment (usavkov-epam) [reply src/common/POLFields/DetailsFields/FieldAcquisitionMethod.js]: Looks like yes, add `useAcqMethods` hook

**Review by NikitaSedyx — APPROVED** [review]

**Review by AndreiShumski2 — APPROVED** [review]

**Pull request merged**